moesifwsgi/logger_helper.py

```diff
@@ -0,0 +1,135 @@
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
+from .parse_body import ParseBody
+from io import BytesIO
+
+
+class LoggerHelper:
+
+    def __init__(self):
+        self.parse_body = ParseBody()
+        self._parse_headers_special = {
+            'HTTP_CGI_AUTHORIZATION': 'Authorization',
+            'CONTENT_LENGTH': 'Content-Length',
+            'CONTENT_TYPE': 'Content-Type',
+        }
+
+    def parse_request_headers(self, environ):
+        try:
+            for cgi_var, value in environ.iteritems():
+                if cgi_var in self._parse_headers_special:
+                    yield self._parse_headers_special[cgi_var], value
+                elif cgi_var.startswith('HTTP_'):
+                    yield cgi_var[5:].title().replace('_', '-'), value
+        except AttributeError:
+            for cgi_var, value in environ.items():
+                if cgi_var in self._parse_headers_special:
+                    yield self._parse_headers_special[cgi_var], value
+                elif cgi_var.startswith('HTTP_'):
+                    yield cgi_var[5:].title().replace('_', '-'), value
+
+    def request_url(self, environ):
+        return '{0}{1}{2}{3}{4}'.format(
```

Comment:
should it be "/" instead of "-"between strings or do u replace it later?

Reply:
It should be "-" because the initial header would be for example `HTTP_CONTENT_TYPE` and we could convert to `CONTENT-TYPE`